runtime/plugin/netrwPlugin.vim

```diff
@@ -31,6 +32,63 @@ set cpo&vim
 " ---------------------------------------------------------------------
 " Public Interface: {{{1
 
+" Commands Launch/URL {{{2
+" surpress output of command; use bang for GUI applications
+
+" set up redirection (avoids browser messages)
+" by default, g:netrw_suppress_gx_mesg is true
+if get(g:, ':netrw_suppress_gx_mesg', 1)
+  if &srr =~# "%s"
+    let s:redir = printf(&srr, has("win32") ? "nul" : "/dev/null")
+  else
+    let s:redir= &srr .. (has("win32") ? "nul" : "/dev/null")
+  endif
+else
+  let s:redir= ""
+endif
+
+if has('unix')
+  if has('win32unix') " Git Bash provides /usr/bin/start script calling cmd.exe //c
+    " use start //b "" to set void title and avoid ambiguity with passed argument
+    command -complete=shellcmd -nargs=1 -bang Launch
+          \ exe 'silent ! start "" //b ' . trim(<q-args>)  s:redir | redraw!
```

Comment:
is this `//b` correct?

Reply:
[I dug](https://github.com/junegunn/fzf/pull/3798/commits) into the [MSYS2 Path conversion magic docs](https://www.msys2.org/docs/filesystem-paths/) to ensure that `/b` is not misinterpreted as a path `\b`.
However, this assumes the `start` script coming with MSYS2 / Git Bash; I am not sure about Cygwin and maybe an additional case distinction is needed (as it only knows `cygstart`!?). I assumed it essentially superseded by MSYS2, Git Bash and WSL and would await feedback.

Reply:
okay fine. I think you may want to adjust the comment above however. Also the comment mentions `start //b ""` while the code uses `start "" //b`